charts/fpl-case-service/values.aat.template.yaml

```diff
@@ -16,6 +16,11 @@ java:
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: https://idam-api.aat.platform.hmcts.net/o/jwks
     GATEWAY_URL: http://gateway-ccd.aat.platform.hmcts.net
+    SPRING_MAIL_HOST: mta.reform.hmcts.net
+    SPRING_MAIL_PORT: 25
+    SPRING_MAIL_TEST_CONNECTION: false
+    SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: true
+    SPRING_MAIL_PROPERTIES_MAIL_SMTP_SSL_TRUST: mta.reform.hmcts.net
```

Comment:
Hmm I am not sure if host shouldn't be a secret. I don't think giving some of our internal hosts is good thing.

Reply:
yes wasn't sure about the host. Think we should add as a secret? @akomisarek

Reply:
I thought about this no secret necessary in my opinion.